lib/modules/manager/pip-compile/extract.ts

```diff
@@ -95,8 +96,14 @@ export async function extractAllPackageFiles(
         type: 'constraint',
       });
     }
-    // TODO(not7cd): handle locked deps
-    // const lockedDeps = extractRequirementsFile(content);
+    const lockedDeps = extractRequirementsFile(fileContent)?.deps;
+    if (!lockedDeps) {
+      logger.warn(
```

Comment:
```suggestion
      logger.debug(
```

this should normally not happen and i think other managers are also silently ignore that

Reply:
This should not happen especially after we successfully parsed the header.